auth-server/src/model/PostSubscription.ts

```diff
@@ -12,31 +14,23 @@ export default class PostSubscription extends Model {
 	user_id!: number
 	post_id!: number
 
-	static get tableName () {
+	static get tableName (): string {
 		return 'post_subscription';
 	}
 
-	static get idColumn () {
+	static get idColumn (): string {
```

Comment:
same here
```suggestion
	static get idColumn (): number {
```

Reply:
retuning column name return 'id'; which is a string